README.md

```diff
@@ -5,96 +5,144 @@
-    - [Registration](#registration)
-    - [IdP Discovery](#idp-discovery)
-      - [Additional configuration](#additional-configuration)
-    - [OpenID Connect](#openid-connect)
-    - [Smart Card IdP](#smart-card-idp)
-    - [Bootstrapping from a recovery token](#bootstrapping-from-a-recovery-token)
-    - [Feature flags](#feature-flags)
-  - [Events](#events)
-    - [ready](#ready)
-    - [afterError](#aftererror)
-    - [afterRender](#afterrender)
-    - [pageRendered](#pagerendered)
-  - [Building the Widget](#building-the-widget)
-    - [Build and test commands](#build-and-test-commands)
-    - [Local development workflow using `yarn link`](#local-development-workflow-using-yarn-link)
-  - [Browser support](#browser-support)
-  - [Contributing](#contributing)
-
-<!-- /TOC -->
-
-## Getting started
-
-Installing the Okta Sign-In Widget into your project is simple. You can include the Sign-In Widget in your project either directly from the Okta CDN, or by packaging it with your app via our npm package, [@okta/okta-signin-widget](https://www.npmjs.com/package/@okta/okta-signin-widget).
-
-You'll also need:
-
-* An Okta account, called an _organization_ (sign up for a free [developer organization](https://developer.okta.com/signup/) if you need one)
-
-### Using the Okta CDN
+  - [Sample applications](#sample-applications)
+- [SDKs](#sdks)
+  - [Javascript](#javascript)
+  - [Java](#java)
+  - [.Net](#net)
+- [API Reference](#api-reference)
+  - [OktaSignIn](#oktasignin)
+  - [showSignInToGetTokens](#showsignintogettokens)
+  - [showSignInAndRedirect](#showsigninandredirect)
+  - [renderEl](#renderel)
+    - [success](#success)
+  - [hide](#hide)
+  - [show](#show)
+  - [remove](#remove)
+  - [on](#on)
+  - [off](#off)
+  - [authClient](#authclient)
+- [Configuration](#configuration)
+  - [Basic config options](#basic-config-options)
+  - [Username and password](#username-and-password)
+  - [Language and text](#language-and-text)
+  - [Colors](#colors)
+  - [Links](#links)
+    - [Help Links](#help-links)
+    - [Sign Out Link](#sign-out-link)
+  - [Buttons](#buttons)
+    - [Custom Buttons](#custom-buttons)
+    - [Registration Button](#registration-button)
+  - [Registration](#registration)
+  - [IdP Discovery](#idp-discovery)
+    - [Additional configuration](#additional-configuration)
+  - [OpenID Connect](#openid-connect)
+  - [Smart Card IdP](#smart-card-idp)
+  - [Bootstrapping from a recovery token](#bootstrapping-from-a-recovery-token)
+  - [Feature flags](#feature-flags)
+- [Events](#events)
+  - [ready](#ready)
+  - [afterError](#aftererror)
+  - [afterRender](#afterrender)
+  - [pageRendered](#pagerendered)
+- [Building the Widget](#building-the-widget)
+  - [Build and test commands](#build-and-test-commands)
+  - [Local development workflow using `yarn link`](#local-development-workflow-using-yarn-link)
+- [Browser support](#browser-support)
+- [Contributing](#contributing)
+
+## Usage guide
```

Comment:
nit - I feel like this should end up as "Usage Guide" vs "Usage guide".   @jakubvul - do we have a style guide I should refamiliarize myself with?

Reply:
I agree, "Usage Guide" is better.